### ISSUE TITLE
F-Droid dead link from GitHub

### ISSUE THREAD

**Issue by sonalder-darlene** [TheLastProject/Catima#82]
The "Get it on F-Droid" button redirects to a 404 error page.

**Comment by TheLastProject** [comment]
I'm still working to get it published on F-Droid, see https://gitlab.com/fdroid/fdroiddata/-/merge_requests/7629. But I should have probably been clearer about that.

I'll leave this issue open until the app is actually available on F-Droid.

Thanks for your interest and to take the time to report an issue you noticed :)

**Comment by sonalder-darlene** [comment]
Okay maybe a "Soon on F-Droid" should be less confusing.
Thanks for your work :)

**Comment by TheLastProject** [comment]
F-Droid is finally live since a day or two ago so I'm closing this issue. Thanks again for taking the time to report it.

**Comment by IzzySoft** [comment]
@TheLastProject seeing that the latest release is still missing from F-Droid: Do you want me to keep Catima in my repo (faster updates; signature yours and thus enables cross-updates with Play)? That of course would require you to keep attaching APKs to releases.

In most cases, once the F-Droid listing is completed I remove the corresponding app at my end. There are some exceptions, though – especially with apps having a "faster pace at releases" (I e.g. was explicitly asked to keep FairEmail in my repo, as F-Droid matches to build every 3rd or 4th release at best), and I gladly make an exception for Catima too. Choice is yours, you've got the last word on this.

**Comment by TheLastProject** [comment]
It is up to you, really. I don't want to give you any unnecessary work. F-Droid seems to have picked up a release at least but it hasn't appeared in the store yet. I will keep attaching APKs to releases anyway, it's kinda a nice flow for me.

**Comment by IzzySoft** [comment]
OK, guess I'll have to consider how I mark such apps. I have waited migrating from Loyalty until it becomes clear which source to use (to avoid another migration). And users need a clear sign whether apps will stay permanently or be moved, for the same reason. As the F-Droid app will only show one description if an app is available in multiple repos (and I'm not 100% sure which one that will be), that mark will probably be on the website only (as are the library details).

As you keep attaching the APK anyway and thus it's no extra work for you, I'll then keep Catima in my repo – and you can [add another badge](https://gitlab.com/IzzyOnDroid/repo/#if-my-app-is-available-here-do-you-have-a-badge-i-can-use-to-link-to-it) to the Readme. It's no extra work for me either: once it's set up, it maintains itself. Especially with Fastlane, where the authors maintain description and screenshots even :smiley:

Thanks!

**Comment by IzzySoft** [comment]
OK, [app details page](https://apt.izzysoft.de/fdroid/index/apk/me.hackerchick.catima) now includes a link if a corresponding marker was set in the app's metadata. Thanks again, so I can start migration to your app now!

**Comment by TheLastProject** [comment]
Sorry, I got kinda caught up in Shantae and the Seven Sirens today and forgot to post here after adding the IzzyOnDroid badge. I'm not sure what you mean about a link and marker in the metadata, but it sounds like good news?

**Comment by IzzySoft** [comment]
Thanks for adding the badge! Could you please fix the link behind it? The URL somehow has a trailing quote included: `<a href="https://apt.izzysoft.de/fdroid/index/apk/me.hackerchick.catima&quot;"`

**Comment by TheLastProject** [comment]
Oops, typo. Fixed, thanks.

**Comment by IzzySoft** [comment]
> I'm not sure what you mean about a link and marker in the metadata, but it sounds like good news?

Yes, I'd say so. And what I mean is the little box below the app's description [here](https://apt.izzysoft.de/fdroid/index/apk/me.hackerchick.catima) – right above the links on top of the screenshots:

![image](https://user-images.githubusercontent.com/6781438/99741247-f7917000-2ad0-11eb-995b-c75c18e51714.png)

This is controlled by a flag in the [metadata I maintain for each app](https://gitlab.com/IzzyOnDroid/repo/-/wikis/Metadata) (`DoubleList` and `DoubleListRef`). The latter is an URL to "more details" – the former I might expand to a comment used when there is no Ref.

And the details I gave above on the differences between the two repos I've turned into a starter for the new [FAQ](https://gitlab.com/IzzyOnDroid/repo/-/wikis/FAQ) – which again will be expanded, given some time (guess I'll move over some parts of the Readme, for example). If you have other questions you'd wish to see answered there, just mention them (ideally in the issues of that GitLab repo) :smiley:
